shared/qcommon/q_platform.h

```diff
@@ -229,13 +229,17 @@ along with this program; if not, see <http://www.gnu.org/licenses/>.
 // endianness
 // Use compiler builtins where possible for maximum performance
 #include <stdint.h>
-#if !defined(__clang__) && (defined(__GNUC__) || defined(__GNUG__))
-
-// gcc
+#if !defined(__clang__) && (defined(__GNUC__) || defined(__GNUG__)) \
+            && __GNUC__ >=4 && __GNUC_MINOR__ >= 3
```

Comment:
Heh, sorry - I've just realised this will cause other issues! It will only be true if the version is 4.3, or 5.3, or 6.3, but 5.1, or 5.2 won't be true. I think the original way of doing this was probably better now :)